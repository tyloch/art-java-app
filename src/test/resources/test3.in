{"type": "Limit", "order": {"direction": "Sell", "id":1, "price": 14, "quantity":20}}
{"type": "Limit", "order": {"direction": "Buy", "id":2, "price": 14, "quantity":10}}