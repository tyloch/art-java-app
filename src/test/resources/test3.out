{"sellOrders": [{"id": 1, "price": 14, "quantity": 10}], "buyOrders": []}
